core/catkin_ws/src/custom_msgs/srv/ToggleModel.srv

```diff
@@ -0,0 +1,4 @@
+string model_name
```

Comment:
Rename to EnableModel because the interface is enabling and disabling

Reply:
Sounds good -- we'll push that change soon.